src/lib/bootloader/udev_mapping.rb

```diff
@@ -37,9 +37,20 @@ def to_hash
     # @param dev [String] device udev or kernel one like /dev/disks/by-id/blabla
     # @raise when device have udev format but do not exists
     def to_kernel_device(dev)
```

Comment:
The documentation of the function should be updated since /dev/md/r0d2n0s128 is not a udev name (I assume with udev names you mean anything in /dev/disk/by-...